app/templates/base.html

```diff
@@ -34,6 +35,16 @@
 <div class="govuk-width-container {{ containerClasses }}">
   {% block beforeContent %}{% endblock beforeContent %}
   <main class="govuk-main-wrapper {{ mainClasses }}" id="main-content" role="main"{% if mainLang %} lang="{{ mainLang }}"{% endif %}>
+    {% with messages = get_flashed_messages() %}
+        {% if messages %}
+            <div class="govuk-grid-row">
+              <div class="govuk-grid-column-full">
```

Comment:
I predict that if a page opts to use `govuk-grid-column-two-thirds` and this is full page width it will look funny but I suppose looking funny isn't an issue at the moment! 

A way to get around this might be to either `{% include %}` (or macro but comes with some closure/ scope implications) this as a one liner explicitly on pages where it would be expected (in the context of their column layout) or to have a standard two thirds column layout in the base that superclasses the `content` block meaning that content is consistent by default unless a page overrides it

Neither feel necessary right now just thinking through how this might look if we wanted to use this pattern longer term

Reply:
Sorry that this went in before you commented 🙏 

Good thoughts, plenty to think about if/when we polish this up more.